blocks/rich-text/index.js

```diff
@@ -696,6 +696,10 @@ export class RichText extends Component {
 			this.props.tagName === prevProps.tagName &&
 			this.props.value !== prevProps.value &&
 			this.props.value !== this.savedContent &&
```

Comment:
Just wondering: are these values ever strict equal? `this.savedContent` seems to change all the time, so it will never be strictly equal to the `value` prop? I'm also having a hard time understanding why we need to check three different values here.

Reply:
I think the idea is that before calling `this.props.onChange` we save the value in `this.savedContent` that way when the component is rerendered with the newly provided `value` prop. this check will fail `this.props.value !== this.savedContent` and we avoid calling `updateContent` (happens when you type continuously on a rich text element)